### PR TITLE
JOUR-2022 - Journeys Refactor

### PR DESCRIPTION
**Pull request by rubinsingh**

### CODE REVIEW

File: src/1_utils.js [hunk 1]
```diff
@@ -353,7 +352,10 @@ utils.cleanLinkData = function(linkData) {
 		safejson.parse(data);
 	}
 	catch (e) {
-		data = goog.json.serialize(data);
+		// no need to serialize for v1/pageview call
```
Comment (ahmednawar) [inline src/1_utils.js]: curious why `/v1/pageview` is different from other endpoints re serialization?

Comment (rubinsingh) [reply src/1_utils.js]: @ahmednawar I've reverted this change in the subsequent commit to make it consistent with the other endpoints. Good call!

**Comment by hemanth-s-a** [comment]
👍  Once circle passes.

**Comment by hemanth-s-a** [comment]
👍

**Pull request merged**